### PR TITLE
Adding doc reference snippets to sample code

### PR DESCRIPTION
**Pull request by john0005**
@tyle8552 Can you look this over?

The developers dev site is now referencing snippet code from samples instead of hard coding them on the site. Any sample code I couldn't find I added a file to `resources/docref` to reference that code till a sample comes along and shows that functionality. 

Issue: https://devtopia.esri.com/runtime/java-se-sdk/issues/817

### CODE REVIEW

**Comment by ghost** [comment]
@john0005 Sorry I didn't see this earlier. I'm going to try to setup a meeting to discuss this. We need to make sure we do this right. The text files in docref doesn't look like the right approach to me.

**Comment by john0005** [comment]
@tyle8552 Sounds good just let me know when.

**Comment by ghost** [comment]
Closing this as we have a new plan.

File: src/main/resources/docref/Use_Cloud_And_Servers-Access_ArcGIS_Platform-Default_Authentication_Challenger.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//[DocRef: Name=Use_Cloud_And_Servers-Access_ArcGIS_Platform-Default_Authentication_Challenger
```
Comment (ghost) [inline src/main/resources/docref/Use_Cloud_And_Servers-Access_ArcGIS_Platform-Default_Authentication_Challenger.txt:1]: This one needs to go in Android.

**Pull request closed without merging**